Notify/Notify.Functions/Notify.Functions/NotifyFunctions/Destination/UpdateDestination.cs

```diff
@@ -80,20 +101,39 @@ private static async void updateExistedDocument(dynamic data, ILogger log, BsonD
 
             if (type.Equals(Constants.NOTIFICATION_TYPE_LOCATION))
             {
-                double latitude = Convert.ToDouble(data.location.latitude), longitude = Convert.ToDouble(data.location.longitude);
-                string address = GoogleHttpClient.Instance.GetAddressFromCoordinatesAsync(latitude, longitude, log).Result;
+                if (data.location.longitude == null || data.location.latitude == null)
```

Comment:
Only if both of them are null it should be removed.
If only one of them is null and the other has a value - it's not acceptable, and the response should be 400 - Bad Request.